x/validatorreward/keeper/distribute.go

```diff
@@ -48,5 +48,8 @@
 
 	k.SetLastDistributeTime(ctx, ctx.BlockHeader().Time)
 
-	k.bankKeeper.SendCoinsFromModuleToModule(ctx, types.ModuleName, types.ValidatorRewardCollectorName, rewardAmount)
+	err := k.bankKeeper.SendCoinsFromModuleToModule(ctx, types.ModuleName, types.ValidatorRewardCollectorName, rewardAmount)
+	if err != nil {
+		panic(err)
```

Comment:
This is ok. Because in [cosmos sdk they use panic too](https://github.com/tokenize-titan/cosmos-sdk/blob/3769645cd186a18a1e6be884bdd94026248aa22b/x/distribution/keeper/allocation.go#L23).